.mailmap

```diff
@@ -0,0 +1,122 @@
+Gabriel Linden Sagula <gsagula@gmail.com>
+Itamar Turner-Trauring <itamar@itamarst.org>           <itamar@datawire.io>
+John Esmet <john.esmet@gmail.com>                      <jesmet@appnexus.com>
+John Esmet <john.esmet@gmail.com>                      <johnesmest@datawire.io>
+John Esmet <john.esmet@gmail.com>                      <johnesmet@datawire.io>
+Luke Shumaker <lukeshu@lukeshu.com>                    <luke@lukeshu.com>
+Luke Shumaker <lukeshu@lukeshu.com>                    <lukeshu@datawire.io>
+Noah Krause <krausenoah@gmail.com>
+Noah Krause <krausenoah@gmail.com>                     <nkrause@MacBook-Pro.local>
+Noah Krause <krausenoah@gmail.com>                     <nkrause@Opss-MacBook-Air.local>
+Noah Krause <krausenoah@gmail.com>                     <nkrause@datawire.io>
+Noah Krause <krausenoah@gmail.com>                     <noahkrause@Noahs-MacBook-Pro.local>
+Philip Lombardi <plombardi89@gmail.com>
+Philip Lombardi <plombardi89@gmail.com>                <893096+plombardi89@users.noreply.github.com>
+Philip Lombardi <plombardi89@gmail.com>                <plombardi@datawire.io>
+Shahriar Rostami <srdsecond@gmail.com>                 <54422255+alphashr@users.noreply.github.com>
+Stephanie Coyle <stephaniecoyle391@gmail.com>
+Stephanie Coyle <stephaniecoyle391@gmail.com>          <55929576+scoyle391@users.noreply.github.com>
+Stephanie Coyle <stephaniecoyle391@gmail.com>          <scoyle@datawire.io>
+
+# Outside contributors (inconsistent email -- did I choosed the correct email?)
+Brandon Catcho <bcatcho@doubledutch.me>                <bcatcho@gmail.com>
+Christoph Grossegger <christoph.grossegger@gmail.com>  <christoph@dust-bit-games.com>
+Dan Sipple <dsipple@etsy.com>                          <sipple.dan@gmail.com>
+Endre Czirbesz <endre.czirbesz@rungway.com>            <endre@czirbesz.hu>
+Endre Czirbesz <endre.czirbesz@rungway.com>            <endrec@users.noreply.github.com>
+Giovanni Gargiulo <gargiulo.gianni@gmail.com>          <giovanni.gargiulo@hbc.com>
+John Morrissey <jmorrissey@devoted.com>                <jwm@devoted.com>
+Josue Diaz <josue_diaz@affirmednetworks.com>           <josuesdiaz15@msn.com>
+Markus Maga <markus@maga.nu>                           <markus@maga.se>
+Niko Kurtti <niko.kurtti@interdax.com>                 <niko@kurtti.eu>
+Phil Peble <philpeble@gmail.com>                       <ppeble@activecampaign.com>
+Romario Maxwell <maxwellra@jncb.com>                   <romario.maxwell@gmail.com>
+Wenhua Zhao <whzhao@gmail.com>                         <wenhuaz@splunk.com>
+n1koo <niko.kurtti@interdax.com>                       <niko@kurtti@eu>
+
+# Outside contributors (inconsistent name)
+Kevin Dagostino <kevin@tonkworks.com>
+Kowalczyk Bartek <bkowalczyyk@gmail.com>
+Mohit Sharma <imoisharma@icloud.com>
+
+# Outside contributors (just @users.norely.github.com substitutions)
+Jordan Neufeld <jordan.neufeld@nfl.com>                <neufeldtech@users.noreply.github.com>
```

Comment:
Also probably worth checking: I thought Nuefeld Tech tried to be careful about the distinction between them and the NFL? Might be misremembering, of course.

Reply:
The `@nfl.com` email is the only one he's ever committed with other than the `@users.noreply.github.com`.